Demos/FCFS/Allocator/FCFSAllocator.py

```diff
@@ -145,6 +146,7 @@ def allocate(self, agents: List["Agent"], environment: "Environment", tick: int)
         """
         astar = AStar(environment, self.bid_tracker, tick)
         allocations: Dict["Agent", "Allocation"] = {}
+        random.shuffle(agents)
```

Comment:
what is this shuffle for?

Reply:
Agents in FCFS should be allocated in random order and not in the order of their owners